src/main/java/core/basesyntax/operationstrategy/BalanceOperationHandler.java

```diff
@@ -0,0 +1,16 @@
+package core.basesyntax.operationstrategy;
+
+import core.basesyntax.model.TransactionDto;
+
+public class BalanceOperationHandler implements OperationHandler {
+    @Override
+    public Integer doOperation(TransactionDto fruitRecord) {
```

Comment:
I'd stick to either FruitRecord or TransactionDTO. In this particular line I misunderstand variable and its type.

Reply:
Done